<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dicom_Viewer</class>
 <widget class="QDialog" name="Dicom_Viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>544</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dicom_Viewer</string>
  </property>
  <widget class="QPushButton" name="Browse_Button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>490</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Browse Image</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="Axial_Plane">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="Sagittal_Plane">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="Coronal_Plane">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="Diagonal_Plane">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>290</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Axial_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Axial Plane:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Sagittal_Label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sagittal Plane:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Coronal_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Coronal Plane:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Diagonal_Label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Diagonal Plane:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
